includes/layouts/cso_header_layout.php

```diff
@@ -210,19 +211,19 @@
                 <ul class="list-unstyled components">
                     <p>Navigation</p>
                     <li class="active">
-                        <a href="#menu" >Add new Member</a>
+                        <a href="/ccportal-master/web/cso_area/create_new_member.php" >Add new Member</a>
```

Comment:
@sempermort this links don't work edit your folder from /ccportal-master to ccportal.. or use ../../